<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WindowPer</class>
 <widget class="QDialog" name="WindowPer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1172</width>
    <height>625</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>1172</width>
    <height>625</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1172</width>
    <height>625</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>:/img/LogoB</normaloff>:/img/LogoB</iconset>
  </property>
  <widget class="QLabel" name="BG">
   <property name="geometry">
    <rect>
     <x>-10</x>
     <y>0</y>
     <width>1172</width>
     <height>625</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="settingBtn">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>60</width>
     <height>60</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="Resours/Recours.qrc">
     <normaloff>:/icon/setting</normaloff>:/icon/setting</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>60</width>
     <height>60</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="exitBtn">
   <property name="geometry">
    <rect>
     <x>1040</x>
     <y>0</y>
     <width>131</width>
     <height>60</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string>Exit</string>
   </property>
   <property name="icon">
    <iconset resource="Resours/Recours.qrc">
     <normaloff>:/icon/icons8-power-off-button-96.png</normaloff>:/icon/icons8-power-off-button-96.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>50</width>
     <height>50</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="deleteAccBtn">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>560</y>
     <width>371</width>
     <height>48</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string>Delete your Account</string>
   </property>
   <property name="icon">
    <iconset resource="Resours/Recours.qrc">
     <normaloff>:/icon/trash</normaloff>:/icon/trash</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>50</width>
     <height>50</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="userSearchBtn">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>500</y>
     <width>180</width>
     <height>48</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>BusyCursor</cursorShape>
   </property>
   <property name="text">
    <string>Search</string>
   </property>
   <property name="icon">
    <iconset resource="Resours/Recours.qrc">
     <normaloff>:/icon/icons8-evidence-96.png</normaloff>:/icon/icons8-evidence-96.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>50</width>
     <height>50</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="addBtn">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>150</y>
     <width>180</width>
     <height>48</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string>Add</string>
   </property>
   <property name="icon">
    <iconset resource="Resours/Recours.qrc">
     <normaloff>:/icon/icons8-plus-math-96.png</normaloff>:/icon/icons8-plus-math-96.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>50</width>
     <height>50</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="logoutBtn">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>500</y>
     <width>180</width>
     <height>48</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string>Logout</string>
   </property>
   <property name="icon">
    <iconset resource="Resours/Recours.qrc">
     <normaloff>:/icon/icons8-arrow-pointing-left-96.png</normaloff>:/icon/icons8-arrow-pointing-left-96.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>50</width>
     <height>50</height>
    </size>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="tweetTxt">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>150</y>
     <width>931</width>
     <height>171</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="saveBtn">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>280</y>
     <width>180</width>
     <height>48</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
   <property name="icon">
    <iconset resource="Resours/Recours.qrc">
     <normaloff>:/icon/icons8-save-96.png</normaloff>:/icon/icons8-save-96.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>50</width>
     <height>50</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="cancelBtn">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>340</y>
     <width>180</width>
     <height>48</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QPushButton" name="nextBtn">
   <property name="geometry">
    <rect>
     <x>970</x>
     <y>340</y>
     <width>180</width>
     <height>48</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string>Next</string>
   </property>
   <property name="icon">
    <iconset resource="Resours/Recours.qrc">
     <normaloff>:/icon/icons8-chevron-right-48.png</normaloff>:/icon/icons8-chevron-right-48.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>40</width>
     <height>40</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="preBtn">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>340</y>
     <width>180</width>
     <height>48</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string>Previous</string>
   </property>
   <property name="icon">
    <iconset resource="Resours/Recours.qrc">
     <normaloff>:/icon/icons8-chevron-left-96.png</normaloff>:/icon/icons8-chevron-left-96.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>40</width>
     <height>40</height>
    </size>
   </property>
  </widget>
  <widget class="QLabel" name="likeTxt">
   <property name="geometry">
    <rect>
     <x>416</x>
     <y>340</y>
     <width>211</width>
     <height>34</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>FreeSerif</family>
     <pointsize>13</pointsize>
     <italic>true</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Likes:</string>
   </property>
  </widget>
  <widget class="QLabel" name="mentionTxt">
   <property name="geometry">
    <rect>
     <x>680</x>
     <y>340</y>
     <width>251</width>
     <height>34</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>FreeSerif</family>
     <pointsize>13</pointsize>
     <italic>true</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Mentions:</string>
   </property>
  </widget>
  <widget class="QLabel" name="followersTxt">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>100</y>
     <width>261</width>
     <height>34</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>FreeSerif</family>
     <pointsize>16</pointsize>
     <italic>true</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Followers:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="editBtn">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>210</y>
     <width>180</width>
     <height>48</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string>Edit</string>
   </property>
   <property name="icon">
    <iconset resource="Resours/Recours.qrc">
     <normaloff>:/icon/icons8-edit-96.png</normaloff>:/icon/icons8-edit-96.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>50</width>
     <height>50</height>
    </size>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>90</y>
     <width>261</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>Followings</string>
   </property>
   <property name="textFormat">
    <enum>Qt::RichText</enum>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="Resours/Recours.qrc"/>
 </resources>
 <connections/>
</ui>
